src/main/java/io/stargate/sgv2/jsonapi/service/embedding/operation/EmbeddingProvider.java

```diff
@@ -175,9 +175,15 @@ protected void checkEmbeddingApiKeyHeader(String providerId, Optional<String> ap
    * @param batchId - Sequence number for the batch to order the vectors.
    * @param embeddings - Embedding vectors for the given text inputs.
    */
-  public record Response(int batchId, List<float[]> embeddings) {
+  public record Response(
+      int batchId, List<float[]> embeddings, VectorizeUsageInfo vectorizeUsageInfo) {
     public static Response of(int batchId, List<float[]> embeddings) {
-      return new Response(batchId, embeddings);
+      return new Response(batchId, embeddings, new VectorizeUsageInfo(0, 0, 0, "", ""));
```

Comment:
Added default VectorizeUsageInfo until all embedding provider class get refactored.

Reply:
Adapted the existing code to avoid compilation error in all the embedding providers. Will remove the `of` method for this record.